Us3 page d accueil

Kuroshin-28 opened the pull request:
Si j'ai fais de la merde :
1. Me dire : "Baptiste ! T'as fais de la merde !"
2. Accepter la Pull Request
3. Me demander de changer mon pseudo Discord en "J'ai fait de la merde 💩 "

Kuroshin-28 committed:
First version of the slider :
- Take 3 elements
- Quick navigation with id link (also works with directionnal arrow)
- Temporary images folder for slider

Kuroshin-28 committed:
Add some content :
- Add ID and class to carrousel element to better integrate navigation
- Using ID control based on total element
- Need to fix dynamic navigation on carousel
- Add some CSS modification due to some class and ID addition
- Add comment in HTML for future integration

Kuroshin-28 committed:
Make some minor adjustment :
- Rename all occurences of "carrousel" to "carousel"
- Delete a problematic comment

Kuroshin-28 committed:
Multiple modification :
- Add navigation dot below carousel and change CSS
- Description text added : need to change it depending on carousel
  element displayed
- Remove "Show More" function for long text.
- Need to figure out how to break a setTimeout function taking manual
  control over carousel

Kuroshin-28 committed:
Carousel manual update :
- Add function to reset timeout function
- Index modification within new function

Kuroshin-28 committed:
Few modifications :
- Modify generic text with a sample of the website description (subject
  to change)
- Make some adjustment in HTML structure
- Change some CSS

Kuroshin-28 committed:
Remove some commentaries.

Kuroshin-28 committed:
Merge branch 'develop' into 'US3-Page_d_Accueil'

Kuroshin-28 committed:
Add modification to carousel :
- Vertical scroll issue solved
- Add accueil service
- Api call in accueil page
- Dynamic display in html

Kuroshin-28 committed:
Multiple modifications :
- Add "Tap" event for carousel link so that it work on phone device
- Modify CSS for better integration and visibility on phone device
- Add a feature to scroll back to carousel section if the links are
  triggered manually
- Remove some code in comments that will no longer be used

Kuroshin-28 committed:
Merge branch 'develop' into 'US3-Page_d_Accueil'

Kuroshin-28 committed:
Emergency modifications :
- Remove "useless" commentaries
- Delete temp/ folder and all files within it

Matthieu28 reviewed (approved):
max-width est en px 👀

bastienapp commented on src/app/components/carousel/carousel.component.ts:
```diff
@@ -0,0 +1,46 @@
+        this.Index = 1;
+      }
+      this.manualClick(this.Index);
+    }, 5000);
```
ça serait bien de mettre les 5000 ms dans une constante, ex : 

private readonly SLIDER_TIME_MS = 5000;

bastienapp commented on src/app/app.module.ts:
```diff
@@ -23,6 +24,8 @@ import { CarouselCardComponent } from "./components/carousel-card/carousel-card.
 @NgModule({
   declarations: [
     AppComponent,
+    AccueilPageComponent,
```
ça serait bien de changer le nom du fichier et des appels en anglais, ex : HomeComponent

Kuroshin-28 committed:
Apply "practical" modifications :
- Rename the "accueil" folder into "home" folder
- Rename all files within "home" folder
- Redo all imports within those files
- Modify app-routing module to include the renamed page
- Modify imports and variable call in app.module
- Minor change into home.service to match the newly renamed page "home"
- No more 'Frenchies' variable/name used in both "carousel" component and
  "home" page/service

bastienapp reviewed (approved):
Top, je valide :+1:

bastienapp commented:
Attention, tu as fermé la Pull Request sans la fusionner (c'était peut-être voulu ?)

Kuroshin-28 commented:
Oui, j'ai encore fais de la merde pour changer, j'ai laissé des commentaires dans le code.

Kuroshin-28 committed:
Minor modifications :
- All variable in carousel component switched to private
- Remove all commentaries from the code edited within US3-Page_d_Accueil
  branch
